qxf2/qxf2-page-object-model
# Fix xpath utililty file

The Xpath utility file has a very poor readability with a lot of nested blocks and  a pylint score of 1. In this PR I have fixed the utility script to improve readability and have got the Pylint score to 10

**To test:**
- Run the script utils/xpath_util.py and make sure it's working fine
- Run a pylint check on it to make sure the score is 10/10
